src/attr/exceptions.py

```diff
@@ -55,3 +55,20 @@ class PythonTooOldError(RuntimeError):
 
     .. versionadded:: 18.2.0
     """
+
+
+class NotCallableError(TypeError):
+    """
+    A ``attr.ib()`` requiring a callable has been set with a value
+    that is not callable.
+
+    .. versionadded:: 19.2.0
```

Comment:
Questions =)

1) Is this 'versionadded' supposed to be the next version?
2) Does it need a changelog entry?

Reply:
yes and yes!